solidity/contracts/bridge/Bridge.sol

```diff
@@ -53,106 +54,24 @@ import "../bank/Bank.sol";
 ///         balances in the Bank.
 /// @dev Bridge is an upgradeable component of the Bank.
 ///
-/// TODO: All wallets-related operations that are currently done directly
-///       by the Bridge can be probably delegated to the Wallets library.
-///       Examples of such operations are main UTXO or pending redemptions
-///       value updates.
+// TODO: All wallets-related operations that are currently done directly
+//       by the Bridge can be probably delegated to the Wallets library.
+//       Examples of such operations are main UTXO or pending redemptions
+//       value updates.
```

Comment:
Is this TODO still relevant?

Reply:
I think not but I wanted to remove it in the final cleanup PR.

Reply:
Removed in https://github.com/keep-network/tbtc-v2/pull/211.